### PR TITLE
twitter bot: v1

### PR DESCRIPTION
**Pull request by ric96**
Version 1 of twitter bot for 96Boards

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -55,6 +55,9 @@
 #    break
 
 # print(delhi,erode,camb,sd,ist_dt.hour)
+
+########################################################################
+
 while True:
  while True:
```
Comment (Mani-Sadhasivam) [inline bot.py]: Nested `while true` smells like the code needs to be structured better ;-)

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Couple of points:

1. Please always add a commit message
2. Only split out the commits if different functionalities are covered otherwise a single commit would be the best

File: bot.py (hunk 1 above)
Comment (ric96) [reply bot.py]: yea just fixed that

**Comment by ric96** [comment]
@Mani-Sadhasivam made the changes as requested

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+  - Create a developer account at: [https://developer.twitter.com/en/apply/user](https://developer.twitter.com/en/apply/user)
+  - Get access token: [https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html](https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html)
+  - Replace <consumer_key> in ```bot.py``` with your Twitter Consumer Key.
+  - Replace <consumer_secret> in ```bot.py``` with your Twitter Consumer Secret Key.
+  - Replace <access_token> in ```bot.py``` with your Twitter access token.
+  - Replace <access_token_secret> in ```bot.py``` with your Twitter access token secret key.
+
+## 2.3) Running the Bot
+
+- **Test Run**
```
Comment (Mani-Sadhasivam) [inline README.md]: Nit: How about:

#### Test Run
 You can run it simply as
```shell
  $ python3 bot.py
```
#### Deploy
From Shell
```shell
  $ python3 bot.py &
```

File: bot.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import tweepy
```
Comment (Mani-Sadhasivam) [inline bot.py:7]: Please add a license here!

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Couple of nitpicks

File: bot.py (hunk 3 above)
Comment (ric96) [reply bot.py:7]: fixed

**Review by Mani-Sadhasivam — APPROVED** [review]
LGTM!

**Pull request merged**